### PR TITLE
Merge Jisho into master branch

### PR DESCRIPTION
**Pull request by M157q**

### CODE REVIEW

File: zdict/dictionaries/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 from .yahoo import YahooDict as yahoo
 from .moe import MoeDict as moe
 from .urban import UrbanDict as urban
+from .jisho import JishoDict as jisho
```
Comment (iblis17) [inline zdict/dictionaries/__init__.py]: Would you make the import in alphabat order?

File: zdict/dictionaries/jisho.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    # Change the url below to the API url of the new dictionary.
+    # Need to keep the `{word}` for `_get_url()` usage.
+    API = 'http://jisho.org/api/v1/search/words?keyword={word}'
+
+    @property
+    def provider(self):
+        # Change `template` to the short name of the new dictionary.
+        return 'Jisho'
+
+
```
Comment (iblis17) [inline zdict/dictionaries/jisho.py]: unnecessary blank line between class method.

File: zdict/dictionaries/jisho.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    def _get_url(self, word) -> str:
+        return self.API.format(word=word)
+
+
```
Comment (iblis17) [inline zdict/dictionaries/jisho.py]: Same as previous remark

**Comment by iblis17** [comment]
Hmm... should we introduce `pep8` format checker in ci build?

File: zdict/dictionaries/__init__.py (hunk 1 above)
Comment (M157q) [inline zdict/dictionaries/__init__.py]: https://github.com/zdict/zdict/pull/77/commits/9eba55da14db2b690075c886470475922e076961

**Comment by iblis17** [comment]
hmm... maybe rebase this branch to master will solve the ci issue.

**Comment by M157q** [comment]
@iblis17 y, same thought here. Should we close this PR and checkout this branch to another branch called jisho-rebase and send the PR again?

**Comment by iblis17** [comment]
You can do force push if no one working on this branch. Otherwise, make a new PR!

**Comment by M157q** [comment]
force pushed.

**Comment by M157q** [comment]
+ unit-tests and better error handling needed.
+ And it stores lots of info from each query but doesn't have verbose mode output.
Maybe we can merge this PR first and add these two above to issues?

**Comment by iblis17** [comment]
I'm fine with both. Go ahead!

**Comment by M157q** [comment]
All checks have passed!

**Comment by M157q** [comment]
close #31

**Comment by M157q** [comment]
I forgot to update README. LOL

**Pull request merged**